build.gradle

```diff
@@ -1,9 +1,18 @@
 buildscript {
     repositories {
-        mavenCentral()
+        jcenter()
```

Comment:
Without a citation, I see no reason to switch. And I trust Maven central more, personally.